src/helpers/db/populate-db.js

```diff
@@ -48,105 +48,118 @@ const collectionPestDistribution = 'PEST_DISTRIBUTION'
-        )
-        await loadData(
-          filePathPestFCPD,
-          mongoUri,
-          server.db,
-          collectionPestFCPD,
-          1
-        )
-        await loadData(
-          filePathPestPras,
-          mongoUri,
-          server.db,
-          collectionPestPras,
-          1
-        )
-
-        await server.start()
-      } catch (error) {
-        logger.error(error)
-      }
+      server.route({
+        method: 'POST',
+        path: '/populateDb',
+        handler: async (request, h) => {
+          try {
+            await loadData(
+              filePathPlant,
+              mongoUri,
+              server.db,
+              collectionNamePlant,
+              2
+            )
+            await loadData(
+              filePathService,
+              mongoUri,
+              server.db,
+              collectionNameServiceFormat,
+              1
+            )
+            await loadData(
+              filePathCountry,
+              mongoUri,
+              server.db,
+              collectionNameCountry,
+              1
+            )
+            await loadData(
+              filePathServiceAnnex6,
+              mongoUri,
+              server.db,
+              collectionNamePlantAnnex6,
+              1
+            )
+            await loadData(
+              filePathServiceAnnex11,
+              mongoUri,
+              server.db,
+              collectionNamePlantAnnex11,
+              1
+            )
+            await loadData(
+              filePathServicePestName,
+              mongoUri,
+              server.db,
+              collectionNamePestName,
+              1
+            )
+            // Load PLANT DATA - COMBINED - START
+            await loadCombinedDataForPlant(
+              mongoUri,
+              server.db,
+              collectionNamePlantName,
+              1
+            )
+            // Load PLANT DATA - Combined - END
+
+            // Load PEST_LINK DATA - START
+            await loadCombinedDataForPestLink(
+              mongoUri,
+              server.db,
+              collectionNamePlantPestLink,
+              1
+            )
+            // Load PEST_LINK DATA - END
+            await loadData(
+              filePathServicePlantPestReg,
+              mongoUri,
+              server.db,
+              collectionNamePlantPestReg,
+              1
+            )
+            await loadData(
+              filePathPestDistribution,
+              mongoUri,
+              server.db,
+              collectionPestDistribution,
+              1
+            )
+            await loadData(
+              filePathPestFCPD,
+              mongoUri,
+              server.db,
+              collectionPestFCPD,
+              1
+            )
+            await loadData(
+              filePathPestPras,
+              mongoUri,
+              server.db,
+              collectionPestPras,
+              1
+            )
+
+            // await server.start()
```

Comment:
why await server.start() is commented. ?.

Reply:
Is it required in your file? There's already one in async in index.js, function startServer() {
  const server = await createServer()
  await server.start()}

Reply:
Please remove or add back again if everything still works.

Reply:
I think the server should start in createServer (this was already there from CDP) and the processing will happen in the background, I have tested it and didnt see any issues.